ios/LayoutReanimation/REAUIManager.mm

```diff
@@ -64,6 +80,25 @@ - (void)setBridge:(RCTBridge *)bridge
   }
 }
 
+- (void)_removeChildren:(NSArray<UIView *> *)children
+          fromContainer:(UIView *)container
+          withAnimation:(RCTLayoutAnimationGroup *)animation
+{
+  if (_proxyViewRemoval) {
+    [_animationsManager removeChildren:children fromContainer:container];
+  } else if (animation != _reactLayoutAnimationGroup) {
```

Comment:
Can we add a comment here what this piece of code does (which problem it resolves) and why this is necessary?